.github/workflows/ci.yml

```
@@ -192,6 +192,19 @@ jobs:
           name: nuget-packages
           path: ./Bin/Release/ILGPU*.${{ steps.package.outputs.version }}.*nupkg
 
+  publish-nuget-pre-release:
+    if: github.ref == 'refs/heads/master'
+    needs: [check-style, build-and-test]
```

ISSUE: Missing reference to `package`.
```suggestion
    needs: [check-style, build-and-test, package]
```